backend/executor.go

```diff
@@ -404,6 +404,8 @@ loop:
 }
 
 // mustEmbedUnimplementedTaskHubSidecarServiceServer implements protos.TaskHubSidecarServiceServer
+//
+//lint:ignore U1000 because this is a required gRPC method
 func (grpcExecutor) mustEmbedUnimplementedTaskHubSidecarServiceServer() {
```

Comment:
What gRPC wants us to do here is to change `type grpcExecutor struct` and add an embed for `unimplementedTaskHubSidecarServiceServer`. We shouldn't have to implement this method

Reply:
I'm not sure what this means, unfortunately. Can you provide an example of the code change I can make that will allow me to delete this method?

Reply:
I think removing this method should be enough.

`grpcExecutor` already embeds `unimplementedTaskHubSidecarServiceServer`:

https://github.com/microsoft/durabletask-go/blob/7c6a32699e1422c035c4d2a90edde62fddee0086/backend/executor.go#L45

I tried doing it locally but right now this PR isn't building (at least not for me?) with some other really weird issues